### PR TITLE
Correcting unix solutions

### LINKED ISSUE #546: Unix shell solutions with old data files
There are a bunch of solutions for exercises in "Building Tools with the Unix Shell" and "Going Further with the Unix Shell" that haven't been updated from when they used Software Carpentry data files (e.g. `pentane.pdb`, `cubane.pdb`, etc). It looks like the exercises in the chapters have been updated to use our data files (i.e. `dracula.txt`, etc) but not the solutions.

### PR DESCRIPTION
**Pull request by k8hertweck**
Resolves #546

### CODE REVIEW

**Comment by k8hertweck** [comment]
This covers standardization from the three shell chapters. I'll be adding revisions for subsequent exercises/solutions in #554

File: chapters/solutions.Rmd [hunk 1]
```diff
@@ -244,97 +277,117 @@ The file should contain the following lines:
...
+1. No: this sorts by the book title
+2. No: this results in an error because `sort` is being used incorrectly
+3. No: there are duplicate dates in the output because they have not been sorted first
+4. Yes: this results in the output shown below.
+5. No: this extracts the desired data (below), 
+but then counts the number of lines,
+resulting in the incorrect answer.
+
+```
+text
```
Comment (DamienIrving) [inline chapters/solutions.Rmd]: Is this supposed to be ` ```text ` instead of having `text` on the next line?

File: chapters/bash-advanced.Rmd [hunk 2]
```diff
@@ -1152,15 +1152,15 @@ In Exercise \@ref(bash-tools-ex-pipe-construction)
...
+and uses a variation of the code you used earlier to print a list
 of the unique publication years appearing in each of those files separately.
 
 ### Counting names {#bash-advanced-ex-sense-sensibility}
 
 You and your friend have just finished reading *Sense and Sensibility*
 and are now having an argument.  
 Your friend thinks that the elder of the two Dashwood sisters,
-Harriet,
+Elinor,
```
Comment (DamienIrving) [inline chapters/bash-advanced.Rmd:1163]: Well spotted!!

File: chapters/solutions.Rmd (hunk 1 above)
Comment (k8hertweck) [reply chapters/solutions.Rmd]: Yes! Thanks for the catch

**Review by DamienIrving — APPROVED** [review]
This is great! Thanks for working through all these exercises.

The only comment I have other than the minor ones I added in text is that the new "Building Tools with the Unix Shell" chapter needs a project outline in the introductory text. i.e.

We'll be continuing to work in the `zipf` project, which after the previous chapter should contain the following files:

```text
zipf/
└── data
    ├── README.md
    ├── dracula.txt
    ├── frankenstein.txt
    ├── jane_eyre.txt
    ├── moby_dick.txt
    ├── sense_and_sensibility.txt
    ├── sherlock_holmes.txt
    └── time_machine.txt
```

**Comment by k8hertweck** [comment]
Awesome, added that last bit to the bash tools chapter.

**Pull request merged**